atd-vze/src/views/Crashes/Crash.js

```diff
@@ -168,7 +168,7 @@ function Crash(props) {
     data &&
     data?.atd_txdot_crashes.length > 0 &&
     data?.atd_txdot_crashes[0]["location_id"];
-  const notEditingCoords = !isEditingCoords && latitude && longitude;
```

Comment:
This was causing an edge case where not having lat/lon for a crash made the edit map show on load. We want to always start with the non-edit map. You can see the bug in prod here -> https://visionzero.austin.gov/editor/#/crashes/19389730

Reply:
i was questioning how it was supposed to work as well! now that you mention it - the original intention probably was to open the edit map when coordinates were missing. 🤔